From e3d913d5fb9ffbd7a229c4c2d7b392f419e97dc8 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Thu, 1 Nov 2018 17:29:29 +0100
Subject: [PATCH 180/293] xlat: update neighbor_cache_entry_flags

* xlat/neighbor_cache_entry_flags.in (NTF_OFFLOADED): New constant,
introduced by Linux commit v4.13-rc1~157^2~252^2~13.
(NTF_STICKY): New constant, introduced by Linux commit
v4.20-rc1~14^2~392.
(NTF_USE, NTF_SELF, NTF_MASTER, NTF_PROXY, NTF_EXT_LEARNED, NTF_ROUTER):
Add fallback definitions.
* NEWS: Mention this.

Co-Authored-by: Dmitry V. Levin <ldv@altlinux.org>
---
 xlat/neighbor_cache_entry_flags.in | 14 ++++++++------
 2 files changed, 9 insertions(+), 7 deletions(-)

diff --git a/xlat/neighbor_cache_entry_flags.in b/xlat/neighbor_cache_entry_flags.in
index 6a210f5..0f0a83e 100644
--- a/xlat/neighbor_cache_entry_flags.in
+++ b/xlat/neighbor_cache_entry_flags.in
@@ -1,6 +1,8 @@
-NTF_USE
-NTF_SELF
-NTF_MASTER
-NTF_PROXY
-NTF_EXT_LEARNED
-NTF_ROUTER
+NTF_USE		0x01
+NTF_SELF	0x02
+NTF_MASTER	0x04
+NTF_PROXY	0x08
+NTF_EXT_LEARNED	0x10
+NTF_OFFLOADED	0x20
+NTF_STICKY	0x40
+NTF_ROUTER	0x80
-- 
1.7.12.4

